src/cargo/util/toml/embedded.rs

```diff
@@ -66,17 +63,8 @@ pub(super) fn expand_manifest(
         let manifest = toml::to_string_pretty(&manifest)?;
         Ok(manifest)
     } else {
-        // Legacy doc-comment support; here only for transitional purposes
-        let comment = extract_comment(content)?.unwrap_or_default();
```

Comment:
Should there be a message saying we no longer support comment-based manifests if we encounter one?

Reply:
This would require still parsing the doc comment and I feel like its not worth keeping around longer for an unstable feature.